tests/testing_helpers.py

```diff
@@ -314,7 +314,7 @@ def build_config_dictionary(config_keys=None):
         "gcp_zone": None,
         "immediate_checkpoint": False,
         "incoming_wals_directory": "/some/barman/home/main/incoming",
-        "keepalive_interval": 60,
```

Comment:
This doesn't look like an expected change.

Reply:
this was unfortunate! My modification was exactly at the same line and i thought this was retained in the rebase! Sorry, my bad!

Reply:
np :)